docs/commands/context.md

```diff
@@ -19,6 +19,20 @@ menu when hitting a breakpoint.
   instructions to be executed.
 
 
+### Adding custom context panes ###
+
+As well as using the built in context panes, you can add your own custom pane that
+will be displated at each break-like event with all the other panes. Custom panes
+can be added using the api:
+```python
+register_external_context_pane(pane_name, display_pane_function, pane_title_function)
+```
+
+Check the [api](../api.md) documentation to see a full usage of the api. Custom panes
```

Comment:
Let's shorten this. `Check the [api](../api.md) documentation to see a full usage of the api.`. The rest is a repeat of the docs you just linked.